packages/multichain-network-controller/src/utils.ts

```diff
@@ -33,3 +42,47 @@ export function checkIfSupportedCaipChainId(
   // Check if the chain id is supported
   return Object.keys(AVAILABLE_MULTICHAIN_NETWORK_CONFIGURATIONS).includes(id);
 }
+
+/**
+ * Updates a network configuration to the format used by the MultichainNetworkController.
+ * This method is exclusive for EVM networks with hex identifiers from the NetworkController.
+ *
+ * @param network - The network configuration to update.
+ * @returns The updated network configuration.
+ */
+export const updateNetworkConfiguration = (
+  network: NetworkConfiguration,
+): MultichainNetworkConfiguration => {
+  return {
+    chainId: toCaipChainId(
+      KnownCaipNamespace.Eip155,
+      hexToNumber(network.chainId).toString(),
+    ),
+    isEvm: true,
+    name: network.name,
+    nativeCurrency: network.nativeCurrency,
+    blockExplorerUrls: network.blockExplorerUrls,
+    defaultBlockExplorerUrlIndex: network.defaultBlockExplorerUrlIndex || 0,
+  };
+};
+
+/**
+ * Updates a record of network configurations to the format used by the MultichainNetworkController.
+ * This method is exclusive for EVM networks with hex identifiers from the NetworkController.
+ *
+ * @param networkConfigurationsByChainId - The network configurations to update.
+ * @returns The updated network configurations.
+ */
+export const updateNetworkConfigurations = (
```

Comment:
Similarly as above, should this be called `toMultichainNetworkConfigurationsByChainId`?

Reply:
https://github.com/MetaMask/core/pull/5315/commits/832e57e42e523c1261bdc8faf6a5e148588a5acf